jwst/pipeline/calwebb_spec3.py

```diff
@@ -236,7 +249,8 @@ def process(self, input):
                 for cal_array in result:
                     cal_array.meta.asn.table_name = op.basename(input_models.asn_table_name)
                 result = self.outlier_detection(result)
-
+                # interpolate pixels that are nanned before resampling 
```

Comment:
Not sure "nanned" is a legit verb?  I would just say "... that are NaN before ..."

And does it really only work on pixels that are NaN, or any pixel that has DQ=DO_NOT_USE?

Reply:
The routine corrects bad pixels defined as DQ contains DO_NOT_USE, NON_SCIENCE or DATA is NaN.
Updated comment